question28/js/forgotpassword.js

```diff
@@ -0,0 +1,39 @@
+import { validateInputValue } from "./modules/validations";
+import { Chance } from "chance";
+const chance = new Chance();
+
+const submitButton = document.getElementById('js-submit-button');
+const mailInputArea = document.querySelector(".js-email");
+
+
+const checkAllValidity = () => {
+  return document.getElementsByTagName("input").length === document.getElementsByClassName("valid").length;
+}
+
+const toggleDisabledOfSubmitButton = (isValid) => {
+  submitButton.disabled = isValid ? false : true;
+}
```

Comment:
const toggleDisabledOfSubmitButton = (isValid) => {
  submitButton.disabled = isValid ? false : true;
}
```suggestion
const toggleDisabledOfSubmitButton = (isValid) => {
  submitButton.disabled = !isValid
}
```